src/plugins/dashboard/public/dashboard_actions/copy_to_dashboard_modal.tsx

```diff
@@ -88,90 +92,70 @@ export function CopyToDashboardModal({
   const descriptionId = 'copyToDashboardDescription';
 
   return (
-    <EuiFocusTrap clickOutsideDisables={true}>
-      <EuiOutsideClickDetector onOutsideClick={closeModal}>
-        <div
-          role="dialog"
-          aria-modal="true"
-          aria-labelledby={titleId}
-          aria-describedby={descriptionId}
-        >
-          <PresentationUtilContext>
-            <EuiModalHeader>
-              <EuiModalHeaderTitle id={titleId} component="h2">
-                {dashboardCopyToDashboardActionStrings.getDisplayName()}
-              </EuiModalHeaderTitle>
-            </EuiModalHeader>
+    <div role="dialog" aria-modal="true" aria-labelledby={titleId} aria-describedby={descriptionId}>
```

Comment:
Would it make sense to wrap this in an `<EuiModal>` component? https://eui.elastic.co/#/layout/modal

Reply:
The `EuiModal` component comes from the `overlays.openModal` function. If we added one here it would technically be opening two modals.